README.md

```diff
@@ -62,7 +62,9 @@ Alternatively, you can use the `accessibilityLabel` element property to give the
 
 ### Track view navigation
 
-Because React Native offers a wide range of libraries to create screen navigation, by default only manual view tracking is supported. You can manually start and stop a view using the following `startView()` and `stopView` methods.
+Because React Native offers a wide range of libraries to create screen navigation, only manual view tracking is supported. To see RUM sessions populate in Datadog, you need to implement view tracking.
```

Comment:
> only manual view tracking is supported

This part is misleading as we have two libraries available to do automatic view tracking. This is why we started with "by default"

Reply:
```suggestion
Because React Native offers a wide range of libraries to create screen navigation, only manual view tracking is supported by default. To see RUM sessions populate in Datadog, you need to implement view tracking.
```